Vision/README.md

```diff
@@ -11,34 +9,34 @@ Our vision is for a World Wide Web that is more inclusive, and more respectful o
 
 We will improve the fundamental integrity of the Web platform. The Web will not only grow in scope and importance in our lives; it will grow in respect for its users, grow in the trust of its users, grow in its inclusion of all humanity as its users.
 
-Our new vision is built on these twin pillars:
-
-* Our existing pillar, on which we built the web: the enormous potential of technological change.
-* Our new pillar for the years ahead: recognition and embodiment of fundamental values into the architecture of the web.
-
-# Our Values for the world-wide web
-
-These are the core values of the Web:
+# Vision for the World-Wide Web
 
 * The Web is for all humanity.
 * The Web is designed for the good of its users.
 * The Web must be safe for its users.
 * There is one interoperable world-wide Web.
 
-# Our action as a Consortium
+# Vision for the W3C
```

Comment:
This is the Vision of the _W3C_, rather than the web - what role the association serves.  This is constructed from previous text.  (It currently duplicates some text from the mission below, but that can be improved later.)